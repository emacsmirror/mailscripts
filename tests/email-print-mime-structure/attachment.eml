Date: Sun, 24 Nov 2019 21:13:45 -0500
Subject: test message
Message-ID: <attachment@mailscripts.example>
From: Alice <alice@mailscripts.example>
To: Bob <bob@mailscripts.example>
Content-Type: multipart/mixed; boundary="xyz"

--xyz
Content-Type: multipart/alternative; boundary="abc123"

--abc123
Content-Type: text/plain

This is a simple message

--abc123
Content-Type: text/html

<html><head></head><body><p>This is a simple message</p></body></html>

--abc123--
--xyz
Content-Type: text/plain
Content-Disposition: attachment; filename="test.txt"

This is a simple attachment file.
