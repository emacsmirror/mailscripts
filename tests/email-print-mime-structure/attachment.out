└┬╴multipart/mixed 594 bytes
 ├┬╴multipart/alternative 234 bytes
 │├─╴text/plain 26 bytes
 │└─╴text/html 72 bytes
 └─╴text/plain attachment [test.txt] 33 bytes
